apidaze/http/__init__.py

```diff
@@ -18,8 +18,11 @@ def __init__(self, api_key: str, api_secret: str):
         if not self.api_key or not self.api_secret:
             raise ValueError('api_key and api_secret must be provided')
 
+        # environment = 'https://api4.apidaze.io/'
+        environment = 'https://cpaas-api.dev.voipinnovations.com'
```

Comment:
Regardless of the environment, it is an API URL. Therefore, it may be better to name it as `api_url`. Also, we may accept an `api_url` from the Client arguments with a default value of `https://api4.apidaze.io/` as we have such cases. The internal developers may like to consume the SDK with other Apidaze APIs.

Reply:
Good point.